src/session.js

```diff
@@ -46,15 +47,20 @@ export default class Session extends EventEmitter {
         if (withPayload)
             payloadScript = isIFrame ? this._getIFramePayloadScript() : this._getPayloadScript();
 
-        return mustache.render(TASK_TEMPLATE, {
+        var taskScript = mustache.render(TASK_TEMPLATE, {
             cookie:               cookies.replace(/'/g, "\\'"),
             sessionId:            this.id,
+            isFirstPageLoad:      this.pageLoadCount === 0,
             serviceMsgUrl:        serverInfo.domain + '/messaging',
             ie9FileReaderShimUrl: serverInfo.domain + '/ie9-file-reader-shim',
             crossDomainPort:      serverInfo.crossDomainPort,
             payloadScript:        payloadScript,
             referer:              referer
         });
+
+        this.pageLoadCount++;
```

Comment:
As for me, the `isFirstPageLoad` variable means that there were no page loads in this session, and the `this.pageLoadCount === 0` check is ok for me